### PR TITLE
[1LP][RFR] temporary workaround to cleanup httpd semaphores when httpd was killed

### PR DESCRIPTION
**Pull request by izapolsk**

### CODE REVIEW

**Comment by RonnyPfannschmidt** [comment]
basing the 1lp on 7f46aac4f0f39045391a19afcfeb536adc1dfe18 as it looks like a working solution that's gruesome but fabulous

**Comment by RonnyPfannschmidt** [comment]
please RFR once the manual testing of this integration is done

**Comment by dajoRH** [comment]
Lint report for commit ac67fbae4c7611b28dd58cfc7e1a7d2f487cd0db:

`cfme/utils/appliance/__init__.py`:
- :bangbang: Line 528:75: **E127** *continuation line over-indented for visual indent*

Please, rectify these issues :smirk: .


**No commit flaws detected**.

*CFME QE Bot*

**Comment by izapolsk** [comment]
I eventually managed to cope with quoting shell. It works locally. Please review.

File: cfme/utils/appliance/__init__.py [hunk 1]
```diff
@@ -511,6 +511,26 @@ def configure(self, log_callback=None, **kwargs):
 
             ipapp.wait_for_ssh()
 
+            # This is workaround for issue when httpd cannot start after restart because
```
Comment (psav) [inline cfme/utils/appliance/__init__.py]: **REQUIRED:** Please move this out into a separate method and call that method from within the configure.

**Pull request merged**